CODEOWNERS

```diff
@@ -2,4 +2,16 @@
 # This file controls who is tagged for review for any given pull request.
 
 # For anything not explicitly taken by someone else:
-* @open-telemetry/dotnet-approvers
+* @open-telemetry/dotnet-contrib-approvers
+
+src/OpenTelemetry.Contrib.Exporter.Stackdriver/                             @open-telemetry/dotnet-contrib-approvers
+src/OpenTelemetry.Contrib.Extensions.AWSXRay/                               @open-telemetry/dotnet-contrib-approvers @srprash @lupengamzn
+src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/                    @open-telemetry/dotnet-contrib-approvers
+src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/              @open-telemetry/dotnet-contrib-approvers
+src/OpenTelemetry.Contrib.Instrumentation.MassTransit/                      @open-telemetry/dotnet-contrib-approvers
+
+test/OpenTelemetry.Contrib.Exporter.Stackdriver.Tests/                      @open-telemetry/dotnet-contrib-approvers
+test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/                        @open-telemetry/dotnet-contrib-approvers @srprash @lupengamzn
```

Comment:
will have to validate if this is going to work, without both you being already have some permissions to this repo.

Reply:
I'm a bit confused by your comment. Do you want to validate if the code owners work without individuals being given write permission or do you want me to remove my self and Bolu from this file for now?

Reply:
> Do you want to validate if the code owners work without individuals being given write permission 

this!

Reply:
Okay. Then we can merge in this PR and I plan to create one today for adding CHANGELOG to AWS Project so we can test it then. 
However, the github [doc](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) does mention that:
> The people you choose as code owners must have write permissions for the repository. When the code owner is a team, that team must have write permissions, even if all the individual members of the team already have write permissions directly, through organization membership, or through another team membership.